app/database/migration/server/index.ts

```diff
@@ -11,10 +11,22 @@ import {MM_TABLES} from '@constants/database';
 const {SERVER: {
     GROUP,
     MY_CHANNEL,
+    TEAM,
     THREAD,
 }} = MM_TABLES;
 
 export default schemaMigrations({migrations: [
+    {
+        toVersion: 4,
+        steps: [
+            addColumns({
+                table: TEAM,
+                columns: [
+                    {name: 'invite_id', type: 'string'},
```

Comment:
I get why you are adding this field into the database, my only worry here is to have an outdated invite_id, but I can see that not being a massive problem